CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - modules/storage/buckets
   - modules/storage/opensearch
 - added ability for artifact buckets to write events to event bridge.
+- added eks node iam role that all eks nodes will assume on start
```

Comment:
This is in the wrong spot...move to Unreleased

Reply:
Ack. Will do for this and next comment.